jetty-ee10/jetty-ee10-maven-plugin/src/test/java/org/eclipse/jetty/ee10/maven/plugin/TestJettyStopMojo.java

```diff
@@ -270,22 +286,21 @@ public void testStopWait() throws Exception
         command.redirectOutput(file.toFile());
         command.redirectErrorStream(true);
         command.directory(root.toFile());
-        Process fork = command.start();
+        fork = command.start();
 
         Awaitility.await().atMost(Duration.ofSeconds(5)).until(() -> Files.exists(file));
         AtomicInteger port = new AtomicInteger(-1);
         Awaitility.await().atMost(Duration.ofSeconds(5)).until(() ->
         {
-            Optional<String> tmp = Files.readAllLines(file).stream()
-                    .filter(s -> s.startsWith("STOP.PORT=")).findFirst();
-            if (tmp.isPresent())
+            int retries = 100;
```

Comment:
Looking at this, I don't think you need to implement some retry logic here as `Awaitility.await().atMost(Duration.ofSeconds(5))` is already retrying to evaluate the condition.
If you need can decrease the delay using `pollDelay()`.

Reply:
@olamy The problem is that the ShutdownService may start to write out the log that we're waiting for, but may not have finished writing the line we're waiting for, so we need to keep reading until we're sure we've got all the output.

Reply:
@janbartel yes I understand this but what I mean here. 
Using `Awaitility.await().atMost(Duration.ofSeconds(5)).until` is already retrying the until function, I do not think there is a need to have retry loop in the `until` as Awaitility will retry anymore with a default pool delay and until 5 seconds.